README.md

```diff
@@ -4,10 +4,32 @@ The Bagel CLI is a simple Python command line tool to automatically read and ann
 [BIDS dataset](https://bids-specification.readthedocs.io/en/stable/) 
 so that it can be integrated into the Neurobagel graph.
 
+## Installation
+### Docker
+To run the CLI in a Docker container:
+1. [Install Docker](https://docs.docker.com/get-docker/).
+2. Clone the repository and build the Docker image locally:
+```bash
+git clone https://github.com/neurobagel/bagel-cli.git
+cd bagel-cli
+docker build -t bagel .
+```
+3. `cd` into your local directory containing your phenotypic .tsv file, Neurobagel-annotated data dictionary, and BIDS directory (if present). Run a `bagel` container with the working directory mounted into the container (and set it as the working directory inside the container), and include your CLI command at the end in the following format:
+```bash
+docker run --rm --volume=$PWD:$PWD -w $PWD bagel <CLI command here>
```

Comment:
oh, interesting. is this the solution you've chosen for making the host paths visible inside the container? I like it! I think you should add a quick explanation about that so someone more experienced with Docker doesn't just replace the paths because they think this is just to make it simpler.